src/python/grpcio_tests/tests/unit/data/foo/baz.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The gRPC authors.
```

Comment:
I mixed this file with another on first glance. Can we rename this file to reflect its purpose? E.g., `bar_with_well_known_protos.proto` or `bar_with_wrappers.proto`.